docs/development/adding-a-package-manager.md

```diff
@@ -14,11 +14,23 @@ Common application logic for Renovate, not specific to particular managers, usua
 
 The manager's `index.ts` file supports the following values/functions:
 
-- extractPackageFile
-- extractAllPackageFiles
-- getRangeStrategy (optional)
-- language (optional)
-- supportsLockFileMaintenance (optional)
+| Value/function                | Optional | Async |
+| ----------------------------- | -------- | ----- |
+| `bumpPackageVersion`          | yes      |       |
+| `extractPackageFile`          |          | yes   |
+| `extractAllPackageFiles`      |          | yes   |
```

Comment:
```suggestion
| `extractAllPackageFiles`      | yes      | yes   |
```

The heading for this value/function says:

> ### `extractAllPackageFiles(packageFiles)` (async, optional)

The table should match, I think?

Reply:
The first column is optional and the second async.